### ISSUE TITLE
[FEATURE] Enable custom title

### ISSUE THREAD

**Issue by nikogamulin** [aporia-ai/mlnotify#43]
**Is your feature request related to a problem? Please describe.**
As the library is useful for other processes as well, not only training, it would be useful to enable title modifications

**Describe the solution you'd like**
Implement a function set_title(custom_title) that replaces custom title, (e.g. Waiting my custom process to complete")

**Comment by SnirShechter** [comment]
@nikogamulin 
Hey, do you mean setting the process title and then using it in the webapp and notifications?

**Comment by nikogamulin** [comment]
@SnirShechter Yes. I have been using this tool for other non-training tasks and therefore would like to set more appropriate title

**Comment by hiyyg** [comment]
I also want this feature. Besides, it would be better if custom messages could be displayed.
